packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -386,6 +386,21 @@ export interface AutoScalingGroupProps extends CommonAutoScalingGroupProps {
   readonly initOptions?: ApplyCloudFormationInitOptions;
 }
 
+/**
+ * Properties that reference an external AutoScalingGroup
+ */
+export interface AutoScalingGroupAttributes {
+  /**
+   * AutoScalingGroup's name
+   */
+  readonly autoScalingGroupName: string;
+
+  /**
+   * AutoScalingGroup's role
+   */
+  readonly grantPrincipal: iam.IRole;
```

Comment:
`grantPrincipal` should be named `role` to match the corresponding property of `AutoScalingGroupProps` found [here](https://github.com/aws/aws-cdk/blob/3e9f04dbbd7aadb8ab4394fefd6281f1d6d30fe0/packages/%40aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts#L366).